libraries/AP_EFI/AP_EFI_DroneCAN.h

```diff
@@ -3,7 +3,7 @@
 #include "AP_EFI.h"
 #include "AP_EFI_Backend.h"
 
-#if HAL_EFI_DRONECAN_ENABLED
+#if AP_EFI_DRONECAN_ENABLED
```

Comment:
could probably move this up, and add the #include "AP_EFI_config.h" first to have the same pattern as other